NEW: @W-17100389@: Add in feedback url to help text and log (#1674)
forcedotcom/sfdx-scanner: messages/path-start-util.md

# error.glob-method-conflict

> Path start point %s is invalid: When specifying a method with the #methodName syntax, glob patterns cannot be used.

Reviewer: ```suggestion
Path start point %s is invalid: You can't use glob patterns when specifying a method with the #methodName syntax.
```

Author: Done. Thanks.
